install/install_openadapt.sh

```diff
@@ -14,8 +14,29 @@ RunAndCheck() {
         exit 1
     fi
 }
+
+# Install the required dependencies for the SVG Mixin
+# Specifically for Vtracer and Cairo
+InstallSVGDependencies() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        RunAndCheck "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.55.0" "install rust"
+        RunAndCheck "cargo install vtracer" "install vtracer"
+        
+        # Check the type of chip
+        cpu=$(sysctl machdep.cpu.brand_string)
+        if [[ $cpu == *"Intel"* ]]; then
+            RunAndCheck "brew install cairo" "install cairo for Apple Intel chip"
+        else
+            RunAndCheck "arch -arm64 brew install cairo" "install cairo for Apple Silicon"
+        fi
+    else
+        echo "Unsupported Operating System : $OSTYPE"
+        exit 1
+    fi
+}
+
 [ -d "OpenAdapt" ] && mv OpenAdapt OpenAdapt-$(date +%Y-%m-%d_%H-%M-%S)
-RunAndCheck "git clone https://github.com/MLDSAI/OpenAdapt.git" "clone git repo"
+RunAndCheck "git clone https://github.com/dianzrong/OpenAdapt.git" "clone git repo"
```

Comment:
This should not be merged, right?

Reply:
Oops, I need to review my code better... Addressed in [8b910ea](https://github.com/MLDSAI/OpenAdapt/pull/198/commits/8b910ea4cdf5ef6a2ddd271507f8a868c2b2e0e5)